src/sysext/sysext.c

```diff
@@ -784,6 +890,19 @@ static int merge(Hashmap *images) {
         if (r < 0)
                 return r;
 
+        if (!arg_no_reload && r != 123) {
```

Comment:
in order to keep indentation levels minimal, please change to:

```c
if (r == 123) /* exit code 123 means: didn't do anything */
        return 0;
```

And then continue here and just do `return 1` at the end.